### ISSUE TITLE
[FEATURE] Customizable notification when approaching a monitored device

### ISSUE THREAD

**Issue by Korb** [Myzel394/locus#104]
I suggest adding to the properties of each shares the distance to me at which (or less) the Locus will notify me with the selected sound signal / TTS-voice.

This would be useful for a spouse who receives automatic notification that their spouse will be home soon. Or the opportunity to meet up with a busy buddy who finally ended up in the same neighborhood. Or, when searching in the natural environment, to prevent excessive convergence of neighboring search groups.

Most likely, it will be necessary to add notification frequency settings if the user has not marked the notification as viewed. And, most likely, preventing repeated notifications if, for some time, another device left the notification zone, and then crossed its borders again.

**Comment by Myzel394** [comment]
You can add alarms to shares to be notified when they enter or leave a specific area - is this what you are looking for?

**Comment by Korb** [comment]
Not really. The functionality that you mean calculates the distance of the monitored device to coordinates fixed in time. I propose to calculate the distance to the dynamic point, the center of which is the current device.

Strictly speaking, this is a different type of task for other situations. Interface it can be arranged as one wizard of the steps "specify a point on the map or click here if you want the starting point to be yourself" and "specify the notification radius with a swipe on the map or numerically in meters".
